stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/creation/builder/RepositoryFacadeBuilder.kt

```diff
@@ -53,28 +55,18 @@ internal class RepositoryFacadeBuilder {
     fun build(): RepositoryFacade {
         val config = requireNotNull(defaultConfig)
         val scope = requireNotNull(coroutineScope)
-
         val factory = RepositoryFactory(getChatDatabase(scope), currentUser)
 
-        val userRepository = factory.createUserRepository()
-        val getUser: suspend (userId: String) -> User = { userId ->
-            requireNotNull(userRepository.selectUser(userId)) { "User with the userId: `$userId` has not been found" }
-        }
+        return RepositoryFacade.refreshAndGet(factory, scope, config)
+    }
 
-        val messageRepository = factory.createMessageRepository(getUser)
-        val getMessage: suspend (messageId: String) -> Message? = messageRepository::selectMessage
+    fun initialise() {
```

Comment:
Why do we need an extra method for initialization here? Can't we just use `build`?